<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSI" failures="1" tests="1" name="Default test" time="66.524" errors="0" timestamp="2025-05-29T18:13:55 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="66.079">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at StepActions.stepActions.verifyOrderConfirmationMessage(stepActions.java:123)
at stepDefinition.stepDefinitions.getOrderConfirmationMessage(stepDefinitions.java:78)
at âœ½.get the order confirmation message(file:///C:/Users/manav/miniProject/src/test/resources/features/Project.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
